www/blog/2023/06/19/karmashapes/index.md

```diff
@@ -0,0 +1,790 @@
+
+On their own, each of these datasets is busy, messy, and don’t represent locations across India well. While the PMGSY data represents habitations (the lowest level division of people loosely accounted for by the Indian government) well, the dataset lacks larger villages, towns, and cities. Karmashapes used SEDAC data to fill in most coverage gaps, along with the NGA point data. The two vectorized raster datasets then brought urban boundaries into focus and helped flush out unique location matches or agglomerations where parent-child structures are implemented to showcase the main place of the urban fabric.
+
+All of this was done through dozens of detailed matching steps, edit sessions, refining poor-quality data, cleaning/scrubbing data attributes, and squeezing out every drop of potential from each data source.
+
+![Pre-Karmashapes polygon coverage](images/pre-ks-coverage-1.png "Pre-Karmashapes polygon coverage")
+_(above) Example polygon coverage, WOF (pre-Karmashapes import)_
+
+![Post-Karmashapes polygon coverage](images/post-ks-coverage-1.png "Post-Karmashapes polygon coverage")
+_(above) Example polygon coverage, Karmashapes_
+
+![Pre-Karmashapes polygon coverage](images/pre-ks-coverage-2.png "Pre-Karmashapes polygon coverage")
+_(above) Example polygon coverage, WOF (pre-Karmashapes import)_
+
+![Post-Karmashapes polygon coverage](images/post-ks-coverage-2.png "Post-Karmashapes polygon coverage")
+_(above) Example polygon coverage, Karmashapes_
+
+# Merging into Who’s On First
+
+Before Karmashapes, locality coverage in WOF was mainly represented using publicly available [GeoNames](https://www.geonames.org/) Point data and roughly 1,500 hand-curated polygons for large, populous localities. In WOF, urban and village-level places in India were imported at the locality placetype. Habitations were imported at the neighbourhood placetype.
+
+However, because Karmashapes was developed using several input datasets, each of different vintage, some geographic overlap between town and village polygons is present in the dataset. To solve this overlap issue, WOF relied on Karmashapes attributes and spatial geoprocessing tools to move some villages to the neighbourhood placetype.
+
+If a Karmashape record fell inside a town, for example, it was imported as a WOF neighbourhood. The “urban” administrative level that represents some of India’s largest metropolitan areas was confusing to fit properly into WOF, due to the sprawling and often overly large polygons created during Karmashapes processing steps. In cases where WOF already maintained hand-curated, accurate polygons for large cities, Karmashapes polygons were added as [alt-geometries](https://whosonfirst.org/docs/geometries/alt/) for these places and the existing polygons were maintained as-is.
+
+Below, you’ll find a chart that maps the “core” admin levels in India’s administrative hierarchy to current (and proposed) Who’s On First placetypes.
+
+
+<table>
```

Comment:
Let's convert tables to Markdown so they can render pretty using the WOF blog's CSS